anchore_engine/clients/syft_wrapper.py

```diff
@@ -0,0 +1,37 @@
+import os
+import json
+import shlex
+
+from anchore_engine.utils import run_command, run_command_list, manifest_to_digest, AnchoreException
+from anchore_engine.subsys import logger
+
+def run_syft(image):
+    proc_env = os.environ.copy()
+    cmd = "syft -vv -o json {image}".format(image=image)
```

Comment:
It's possible that some of the command line options for syft may change / adjust in the future, would it be worth making them configurable?

Reply:
I think adding more features as we go (throughout the 3.0 development process) makes sense, not necessarily on this PR quite yet.